bucket/qbittorrent-enhanced.json

```diff
@@ -1,16 +1,12 @@
 {
-    "version": "4.4.2.10",
+    "version": "4.4.3.12",
     "description": "qBittorrent BitTorrent client with anti-leech enhancements",
     "homepage": "https://github.com/c0re100/qBittorrent-Enhanced-Edition",
     "license": "GPL-2.0-or-later",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.4.2.10/qbittorrent_enhanced_4.4.2.10_Qt6_x64_setup.exe#/dl.7z",
-            "hash": "e8cddb2a2847bdedc0d3e07acb688747f9da734a5064a047ea379ddcad4ae2da"
-        },
-        "32bit": {
-            "url": "https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.4.2.10/qbittorrent_enhanced_4.4.2.10_setup.exe#/dl.7z",
-            "hash": "5464423268ffeded3f1ef3346f8bf3960db7ab4b141d5eb01e60faf38db4be9e"
+            "url": "https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.4.3.12/qbittorrent_enhanced_4.4.3.12_Qt6_setup.exe#/dl.7z",
```

Comment:
You are updating the wrong one. This should be in `64bit`, not `32bit`.

Reply:
Eh~ I am sorry, but it is exactly placed in `64bit`. Maybe I understand it wrong?

Reply:
> Eh~ I am sorry, but it is exactly placed in `64bit`. Maybe I understand it wrong?

Sorry I misread it. I should open the raw file to check, instead of reading it in Github's web interface.